auth.py

```diff
@@ -11,6 +11,40 @@
 
 auth = Blueprint('auth', __name__)
 
+def send_verification_email(user):
+    if not user.verified:
+        token = user.get_token(86400)
+        msg = Message('PetAdoption: Email Verification', sender='petadoption.sps@gmail.com',
```

Comment:
there are a couple of undefined variables in this file including `Message`, `account`, `mail`, `update_user`, `RequestVerificationMail`. could you use pylint to check the imports and fix them?